app/en/home/local-deployment/configure/configuration-templates/page.mdx

```diff
@@ -15,115 +15,115 @@ api:
   development: ${env:API_DEVELOPMENT}
   host: ${env:ARCADE_API_HOST}
   port: ${env:ARCADE_API_PORT}
-
+  # Optionally set public_host, in case the Arcade Engine is hosted in a container or behind a reverse proxy, for example
+  public_host: ${env:ARCADE_API_PUBLIC_HOST}
```

Comment:
```suggestion
  # Optionally set public_host, in case the Arcade Engine is hosted in a container or behind a reverse proxy
  #public_host: ${env:ARCADE_API_PUBLIC_HOST}
```

Would rather keep this commented so that a naive copy-paster won't get unexpected behavior. wdyt @byrro?